leakcanary-android-core/src/main/java/leakcanary/internal/DisplayLeakAdapter.kt

```diff
@@ -61,16 +63,16 @@ internal class DisplayLeakAdapter private constructor(
   private val isLeakGroup = instanceProjections.isNotEmpty()
 
   constructor(
-    resources: Resources,
+    context: Context,
```

Comment:
Are we doing all these changes so that we can call ContextCompat.getColor when we definitely don't want themes :) ?

Reply:
if you don't want themes, don't use resources

Reply:
I like when Android Studio shows little color squares on the side.

Reply:
so you're just insinuating that the people who deprecated the non-theme-supporting, color-loading methods didn't take into account people who are hardcoding colors that don't change but also want the little color square?

Reply:
🤷‍♂️😘